freeipa/src/main/java/com/sequenceiq/freeipa/flow/freeipa/verticalscale/event/FreeIpaVerticalScaleEvent.java

```diff
@@ -6,11 +6,11 @@
 public enum FreeIpaVerticalScaleEvent implements FlowEvent {
     STACK_VERTICALSCALE_EVENT,
     STACK_VERTICALSCALE_FINISHED_EVENT(EventSelectorUtil.selector(FreeIpaVerticalScaleResult.class)),
-    STACK_VERTICALSCALE_FINISHED_FAILURE_EVENT(EventSelectorUtil.failureSelector(FreeIpaVerticalScaleRequest.class)),
+    STACK_VERTICALSCALE_FINISHED_FAILURE_EVENT(EventSelectorUtil.failureSelector(FreeIpaVerticalScaleFailureEvent.class)),
 
-    STACK_VERTICALSCALE_FINALIZED_EVENT,
-    STACK_VERTICALSCALE_FAILURE_EVENT(EventSelectorUtil.selector(FreeIpaVerticalScaleFailureEvent.class)),
-    STACK_VERTICALSCALE_FAIL_HANDLED_EVENT;
+    FINALIZED_EVENT("STACKVERTICALSCALEFINALIZEDEVENT"),
+    FAILURE_EVENT("STACKVERTICALSCALEFAILUREEVENT"),
+    FAIL_HANDLED_EVENT("STACKVERTICALSCALEFAILHANDLEDEVENT");
```

Comment:
wouldn't this change for the selector could cause issues for running flows, or failed flows when retry is used?

Reply:
this feature is not even used on dev or everywhere